app/lib/applets/substitutions/substitutions_view.dart

```diff
@@ -43,67 +44,95 @@ class _SubstitutionsViewState extends State<SubstitutionsView> with TickerProvid
+      SubstitutionPlan substitutionPlan, RefreshFunction? refresh) {
     double deviceWidth = MediaQuery.of(context).size.width;
 
     List<Widget> substitutionViews = [];
 
     for (int dayIndex = 0;
-    dayIndex < substitutionPlan.days.length;
-    dayIndex++) {
+        dayIndex < substitutionPlan.days.length;
+        dayIndex++) {
       final int entriesLength =
           substitutionPlan.days[dayIndex].substitutions.length;
 
       substitutionViews.add(RefreshIndicator(
         key: globalKeys[dayIndex + 1],
-        notificationPredicate: refresh != null ? (_) => true : (_) => false, // Hide refresh indicator without bloating the code
+        notificationPredicate: refresh != null
+            ? (_) => true
+            : (_) => false, // Hide refresh indicator without bloating the code
         onRefresh: refresh ?? () async {},
         child: Padding(
           padding: EdgeInsets.only(left: padding, right: padding, top: padding),
-          child: (deviceWidth > 505)
-              ? GridView.builder(
-            itemCount: entriesLength + 1,
-            itemBuilder: (context, entryIndex) {
-              if (entryIndex == entriesLength) {
-                return Padding(
-                  padding: EdgeInsets.only(bottom: padding),
-                  child: lastWidget(
-                      entriesLength: entriesLength,
-                      lastEdit: substitutionPlan.lastUpdated),
-                );
-              }
-
-              return Card(
-                child: SubstitutionGridTile(
-                    substitutionData: substitutionPlan
-                        .days[dayIndex].substitutions[entryIndex]),
-              );
-            },
-            gridDelegate: const SliverGridDelegateWithMaxCrossAxisExtent(
-                maxCrossAxisExtent: 500, childAspectRatio: 20 / 11),
-          )
-              : ListView.builder(
-            itemCount: entriesLength + 1,
-            itemBuilder: (context, entryIndex) {
-              if (entryIndex == entriesLength) {
-                return Padding(
-                  padding: EdgeInsets.only(bottom: padding),
-                  child: lastWidget(
-                      entriesLength: entriesLength,
-                      lastEdit: substitutionPlan.lastUpdated),
-                );
-              }
-
-              return Padding(
-                padding: const EdgeInsets.only(bottom: 8),
-                child: Card(
-                  child: SubstitutionListTile(
-                      substitutionData: substitutionPlan
-                          .days[dayIndex].substitutions[entryIndex]),
-                ),
-              );
-            },
-          ),
+          child: Column(children: [
+            if (_tabController != null &&
+                substitutionPlan.days[dayIndex].infos != null)
+              StreamBuilder(
+                  stream:
+                      accountDatabase.kv.subscribe('enableSubstitutionsInfo'),
```

Comment:
This is the wrong location in the database to look for a value. The "CombinedAppletBuilder" has build in settings that should be used for this.